scripts/lib/setup_venv.py

```diff
@@ -24,7 +24,6 @@
     "libmemcached-dev",
     "python3-dev",          # Needed to install typed-ast dependency of mypy
     "python3-pip",
-    "python-pip",
```

Comment:
@timabbott I am not sure about the history of the different python2 packages here and if they are required as a dependency for installation of any other packages on some OSes.

Reply:
I think we don't need for this anymore.